pyproject.toml

```
@@ -112,10 +113,8 @@ dev = [
     "pytest>=6",
     "pytest-xdist",
     "nbmake",
-]
-# Reading CSV files
-pandas = [
-    "pandas>=1.5.0",
+    # To access the metadata for python packages (in python < 3.10)
+    "importlib-metadata",
```

How about we mark the existence of the dependency only when it's needed? Like this:

```suggestion
    # To access the metadata for python packages
    "importlib-metadata; python_version < '3.10'",
```